numToPair.c

```diff
@@ -0,0 +1,24 @@
+#include "main_prv.h"
+#include "main_declare.h"
+
+extern const int MAX_COLORPAIR_NAME_CHARS;
+/*
+ **********************************************************************************************************************
+ * Service name         : verifyNumberToPair
+ * Syntax               : void verifyNumberToPair(int pairNumber, enum MajorColor expectedMajor, enum MinorColor expectedMinor)
+ * param[in]            : pairNumber
+ * param[in]            : MajorColor
+ * param[in]            : expectedMinor
+ * return               : None
+ * Description          : Program to verify color number with color's pair
+ **********************************************************************************************************************
+ */
+void verifyNumberToPair(int pairNumber, enum MajorColor expectedMajor, enum MinorColor expectedMinor)
```

Comment:
This is `verifyValidNumberToPair`. What about invalid numbers? Neither test nor comment speaks about it.